Source/Fuse.Elements/Resources/FileImageSource.uno

```diff
@@ -251,8 +257,9 @@ namespace Fuse.Resources
 			{
 				try
 				{
-					var data = _file.ReadAllBytes();
```

Comment:
This moves the IO to the calling-thread, doesn't it? If so, I think this is a bad move. Can't we just wrap this in a buffer here instead, like the old code did?